templates/addons/azure-cni-v1/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://raw.githubusercontent.com/Azure/azure-container-networking/master/hack/manifests/cni-installer-v1.yaml
```

Comment:
can we pin this to a release/tag/commit so it doesn't break on us?

Reply:
Pinned it to `v1.5.3`.